splice_machine/src/test/java/com/splicemachine/derby/test/framework/SpliceUnitTest.java

```diff
@@ -526,7 +526,7 @@ protected void testFail(String sqlText,
         }
     }
 
-    protected void testUpdateFail(String sqlText,
+    public static void testUpdateFail(String sqlText,
```

Comment:
This change is not warranted. Make DropTableIT extend SpliceUnitTest and call testUpdateFail().
Use DecimalIT as an example.

Reply:
Done.